From 55231495461c548b7da9a7029016234de4462bb9 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 19 Feb 2018 21:44:02 +0900
Subject: [PATCH 485/909] arm64: dts: r8a77990-ebisu: Enable HS200 of SDHI3

commit f73e5795ec1f7829a36abad447f202f41227ee82 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds the mmc-hs200-1_8v property to SDHI3 and enable eMMC
HS200 mode.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index 0aef33aeb10e..7e4929cfe8f5 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -492,6 +492,7 @@
 
 	vmmc-supply = <&reg_3p3v>;
 	vqmmc-supply = <&reg_1p8v>;
+	mmc-hs200-1_8v;
 	bus-width = <8>;
 	non-removable;
 	status = "okay";
-- 
2.17.1

